libs/QtMate/graph/view.cpp

```diff
@@ -37,7 +37,7 @@ GraphicsView::GraphicsView(QWidget* parent)
 	setResizeAnchor(NoAnchor);
 	setTransformationAnchor(AnchorUnderMouse);
 	setDragMode(QGraphicsView::ScrollHandDrag);
-	setRenderHints(QPainter::Antialiasing | QPainter::TextAntialiasing | QPainter::Antialiasing | QPainter::SmoothPixmapTransform);
```

Comment:
I didn't write the original code, and I don't really master Qt that much, but why remove the antialiasing?
Is it included in another flag?

Reply:
This is not removing, it's de-duplicating. First and third flag are the same.

Reply:
Damn, never review code at the end of a hard week :p